aten/src/ATen/KinetoPluginAPI.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <c10/core/DeviceType.h>
+#include <c10/macros/Macros.h>
+
+#include <libkineto.h>
+
+namespace c10 {
+namespace kineto_plugin {
+
+using PluginFactory = libkineto::ChildActivityProfilerFactory;
+void C10_API insertKinetoPluginRegister(DeviceType t, PluginFactory f);
```

Comment:
can we align to allocate to rename it, say, setKinetoPluginRegister?

Reply:
> can we align to allocate to rename it, say, setKinetoPluginRegister?

yes.